### PR TITLE
doc: session_alias and rows_affected in information_schema.processlist

### PR DESCRIPTION
**Pull request by ekexium**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Update the columns in information_schema.processlist.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

This PR is for v8.3 and later releases. For v8.2, there is another PR https://github.com/pingcap/docs-cn/pull/18181.

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tidb/issues/54486

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: identify-expensive-queries.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ TiDB 会将执行时间超过 [`tidb_expensive_query_time_threshold`](/system-va
 * `table_ids`：语句涉及到的表的 ID。
 * `txn_start_ts`：事务的开始时间戳，也是事务的唯一 ID，可以用这个值在 TiDB 日志中查找事务相关的其他日志。
 * `sql`：SQL 语句。
+* `session_alias`: 当前连接的别名。
```
Comment (qiancai) [inline identify-expensive-queries.md:28]:
```suggestion
* `session_alias`：当前连接的别名。
```

File: information-schema/information-schema-processlist.md [hunk 2]
```diff
@@ -79,6 +83,8 @@ SELECT * FROM processlist\G
 * `DISK` 列：磁盘空间使用情况，单位是 byte。
 * `TxnStart`列：显示事务的开始时间。
 * `RESOURCE_GROUP`列：显示对应的资源组名称。
+* `SESSION_ALIAS`列：显示当前连接的别名。
+* `ROWS_AFFECTED`列：显示当前语句影响的行数。
```
Comment (qiancai) [inline information-schema/information-schema-processlist.md:87]:
```suggestion
* `TxnStart` 列：显示事务的开始时间。
* `RESOURCE_GROUP` 列：显示对应的资源组名称。
* `SESSION_ALIAS` 列：显示当前连接的别名。
* `ROWS_AFFECTED` 列：显示当前语句影响的行数。
```

File: identify-expensive-queries.md [hunk 3]
```diff
@@ -25,6 +25,8 @@ TiDB 会将执行时间超过 [`tidb_expensive_query_time_threshold`](/system-va
 * `table_ids`：语句涉及到的表的 ID。
 * `txn_start_ts`：事务的开始时间戳，也是事务的唯一 ID，可以用这个值在 TiDB 日志中查找事务相关的其他日志。
 * `sql`：SQL 语句。
+* `session_alias`：当前连接的别名。
+* `affected rows`：语句当前影响的行数。
```
Comment (qiancai) [inline identify-expensive-queries.md:29]:
```suggestion
* `affected rows`：当前语句影响的行数。
```

**Review by bb7133 — APPROVED** [review]
LGTM

File: identify-expensive-queries.md (hunk 3 above)
Comment (ekexium) [reply identify-expensive-queries.md:29]: 这个我想强调的意思是，同一个语句，这个值反复查询，是会变的，反映的是接近实时的值。而且这个日志里只有一个语句，不会有歧义。是不是“语句当前”更好？

Comment (qiancai) [reply identify-expensive-queries.md:29]: 那 information-schema/information-schema-processlist.md 这个文档里的写的 “当前语句影响的行数” 需要改成一致的吗

"`ROWS_AFFECTED` 列：显示当前语句影响的行数。"

Comment (ekexium) [reply identify-expensive-queries.md:29]: 可能也改一下更好？可以强调这个信息是近实时的

File: information-schema/information-schema-processlist.md [hunk 4]
```diff
@@ -13,55 +13,59 @@ aliases: ['/docs-cn/dev/information-schema/information-schema-processlist/']
 * `DIGEST` 列：显示 SQL 语句的 digest。
 * `MEM` 列：显示正在处理的请求已使用的内存，单位是 byte。
 * `DISK` 列：显示磁盘空间使用情况，单位是 byte。
-* `TxnStart`列：显示事务的开始时间。
-* `RESOURCE_GROUP`列：显示对应的资源组名称。
+* `TxnStart` 列：显示事务的开始时间。
+* `RESOURCE_GROUP` 列：显示对应的资源组名称。
+* `SESSION_ALIAS` 列：显示当前连接的别名。
+* `ROWS_AFFECTED` 列：显示当前语句影响的行数。
```
Comment (qiancai) [inline information-schema/information-schema-processlist.md:19]:
```suggestion
* `ROWS_AFFECTED` 列：显示语句当前影响的行数。
```

File: information-schema/information-schema-processlist.md [hunk 5]
```diff
@@ -77,8 +81,10 @@ SELECT * FROM processlist\G
 * `DIGEST` 列：SQL 语句的 digest。
 * `MEM` 列：正在处理的请求已使用的内存，单位是 byte。
 * `DISK` 列：磁盘空间使用情况，单位是 byte。
-* `TxnStart`列：显示事务的开始时间。
-* `RESOURCE_GROUP`列：显示对应的资源组名称。
+* `TxnStart` 列：显示事务的开始时间。
+* `RESOURCE_GROUP` 列：显示对应的资源组名称。
+* `SESSION_ALIAS` 列：显示当前连接的别名。
+* `ROWS_AFFECTED` 列：显示当前语句影响的行数。
```
Comment (qiancai) [inline information-schema/information-schema-processlist.md:87]:
```suggestion
* `ROWS_AFFECTED` 列：显示语句当前影响的行数。
```

File: identify-expensive-queries.md (hunk 3 above)
Comment (qiancai) [reply identify-expensive-queries.md:29]: Done in [`798bf63` (#18164)](https://github.com/pingcap/docs-cn/pull/18164/commits/798bf6302b6e23f31737c45100cd3a0b92ae8efc)

File: releases/release-8.2.0.md [hunk 6]
```diff
@@ -184,6 +184,10 @@ TiDB 版本：8.2.0
 | TiKV | [`server.grpc-compression-type`](/tikv-configuration-file.md#grpc-compression-type) | 修改 | 该配置项现在也会影响 TiKV 向 TiDB 发送的响应消息的压缩算法。开启压缩可能消耗更多 CPU 资源。 |
 | TiFlash | [`security.redact_info_log`](/tiflash/tiflash-configuration.md#配置文件-tiflashtoml) | 修改 | 可选值新增 `marker` 选项。当配置项的值设置为 `marker` 时，日志中的用户数据会被标记符号 `‹ ›` 包裹。 |
 
+### 系统表
+
+* 在系统表 [`INFORMATION_SCHEMA.PROCESSLIST`](/information-schema/information-schema-processlist.md) 和 [`INFORMATION_SCHEMA.CLUSTER_PROCESSLIST`](/information-schema/information-schema-processlist.md#cluster_processlist) 中新增 `SESSION_ALIAS` 字段，用于显示当前连接的别名。[#46889](https://github.com/pingcap/tidb/issues/46889) @[lcwangchao](https://github.com/lcwangchao)
```
Comment (qiancai) [inline releases/release-8.2.0.md:189]: this change is also reflected in the v8.2 PR https://github.com/pingcap/docs-cn/pull/18181

**Review by Oreoxmt — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Pull request merged**